packages/fxa-auth-server/lib/senders/emails/templates/recovery/en.ftl

```diff
@@ -1,4 +1,8 @@
 recovery-subject = Reset your password
-recovery-title = Need to reset your password?
-recovery-description = Click the button within the next hour to create a new password. The request came from the following device:
+recovery-title-2 = Forgot your password?
+# Information on the browser, IP address, date and time of the request that
+# triggered the email follows.
+recovery-request-origin = We received a request for a password change on your Firefox account from:
```

Comment:
`We received a request for a password change on your { -product-firefox-account } from:`